src/app.js

```
@@ -1 +1,82 @@
+/* eslint-disable no-console */
+
 // write code here
+const fs = require('fs');
+
+const path = require('path');
+
+function moveFile(src, dst) {
+  let dstFixed = dst;
+
+  const srcFileName = path.basename(src);
+  const dstFolder = path.dirname(dst);
+
+  if (src === undefined || dst === undefined) {
+    console.error('error if only one argument is provided');
+
+    return;
+  }
+
+  if (src === dst) {
+    return;
+  }
+
+  // src must exist
+  if (!fs.existsSync(src)) {
+    console.error('error for non-existent source file');
+
+    return;
+  }
+
+  if (!fs.existsSync(dstFolder)) {
+    console.error('destination path must exist');
+
+    return;
```

According to the task description, if the destination directory does not exist, an error must be thrown. Currently, you're just logging an error message to the console and returning from the function. Instead, you should throw an error to adhere to the task requirements. Also, this check is redundant since you are checking the destination existence in the `if` condition at line 38.